packages/launch/src/utils.ts

```diff
@@ -59,3 +59,12 @@ export const getDomainProduct = (
 		source: flow,
 	},
 } );
+
+export type Product = {
+	product_id: number;
+};
+
+export const isDomainProduct = ( item: Product ): boolean => {
+	const DOMAIN_PRODUCT_ID = 148;
+	return item.product_id === DOMAIN_PRODUCT_ID;
```

Comment:
I'm running under the assumption that all domain products have the ID of 148.
Alternatively, we could check for existence of properties that is specific to a domain product.

Reply:
@daledupreez could you please confirm the assumption about the domain product ID?

Reply:
Unfortunately, we have about 300 product IDs for domains! 😬 This approach is probably not going to work.

It may be worth taking a look at the approach from the `isDomainRegistration()` function in [is-domain-registration.js](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/products-values/is-domain-registration.js). It looks like products in the shopping cart should have an `is_domain_registration` flag.

Reply:
Thanks @daledupreez! Unable to import calypso libs in packages but I looked into how the schema validates and it looks particularly for `is_domain_registration`. 

It does a double check for `isDomainRegistration`, so I was a little puzzled, perhaps it used to be camelCase. So I digged into WPCOM files and I can verify that only `is_domain_registration` is used now.

So the check is simply to check for the property `is_domain_registration`. :)